examples/scenarioHaloOrbit.py

```diff
@@ -183,6 +183,11 @@ def run(showPlots=True):
     viz = vizSupport.enableUnityVisualization(scSim, simTaskName, scObject,
                                               # saveFile=__file__
                                               )
+    viz.settings.showCelestialBodyLabels = 1
```

Comment:
All vizInterface code needs to be behind `if vizSupport.vizFound:` statement in case BSK is compiled without vizInterface.  I pushed a commit using the `[squash]` label that you can integrate with your earlier commit.

Reply:
Resolved, thanks!